content/courses/state-compression/generalized-state-compression.md

```diff
@@ -718,47 +762,47 @@ impl NoteLog {
     // Constructs a new note from given leaf node and message
     pub fn new(leaf_node: [u8; 32], owner: Pubkey, note: String) -> Self {
         Self { leaf_node, owner, note }
-    }
+ }
 }
 ```
 
-In a traditional Anchor program, this would be an account struct, but since
-we’re using state compression, our accounts won’t be mirroring our native
-structures. Since we don’t need all the functionality of an account, we can just
-use the `AnchorSerialize` derive macro rather than the `account` macro.
+In a traditional Anchor program, this would typically be represented by an
+account struct. However, our accounts won't directly mirror our native
+structures because we're using state compression. And since we don't need the
+full functionality of an account, we can use the `AnchorSerialize` derive macro
+instead of the `account` macro.
 
-#### 3. Define input accounts and constraints
+#### 3. Define Input Accounts and Constraints
 
-As luck would have it, every one of our instructions will be using the same
-accounts. We’ll create a single `NoteAccounts` struct for our account
-validation. It’ll need the following accounts:
+In our setup, all instructions will use the same accounts, so we'll create a
```

Comment:
```suggestion
In our setup, all instructions handlers will use the same accounts, so we'll create a
```

Please always refer to the functions that handle instructions as instruction handlers. **Fix this everywhere**. See CONTRIBUTING.md.